### ISSUE TITLE
Bishop with arrows instead of cross

### ISSUE THREAD

**Issue by Little-Magician** [Little-Magician/ShogiGUI-piece-sets#2]
Not everyone is Catholic or Christian. I made bishops with movement arrows to replace the cross because i saw someone saying something similar about hidetchi's bishop.
The arrows however look blurry on shogiGUI.

**Comment by Little-Magician** [comment]
Maybe a whole new piece can be used thats not inspired by the bishop but what?

**Comment by aphirst** [comment]
Maybe you could use the other interpretation of 角, which is something like "animal horn" (interesting btw how that relates to angles, eh) ... "horned chariot", perhaps?

**Comment by Little-Magician** [comment]
Yes I saw hidetchi's video again the one about the origin of the kanjis used on the pieces, maybe I'll rewatch the part where he talks about the bishop and an idea might spark.

**Comment by Little-Magician** [comment]
I discussed with vlad and he helped come up with the idea to use an elephant i think its reasonable since elephants have horns and also its more relate-able to its promotion the dragon horse because both are animals; although, I'm not 100% convinced.

**Comment by aphirst** [comment]
I mean, elephants relate to bishops as far as historical piece developments are concerned, right? Also, elephants are used for the bishops in doubutsu shogi. It should be fine, even though it does seem a bit weird. Mull on it for a while, maybe.

**Comment by Little-Magician** [comment]
I think horns is not a good relationship to the bishop; although, the kanji may have originated from horns it doesn't have any relationship to the current meaning of the kanji which is "angle", but for the meantime I'll use an Elephant for the bishop.
There's also the possibility of using a ninja star since it points in the 4 directions the bishop can move.

**Comment by Little-Magician** [comment]
Elephant has been made but is currently in no set.

**Comment by Little-Mage** [comment]
https://github.com/ornicar/lila/blob/master/COPYING.md
